javascript/javascript3/week2/app.js

```diff
@@ -0,0 +1,69 @@
+let exchangeRates;
+
+const getExchangeData = async () => {
+  const res = await fetch("https://open.er-api.com/v6/latest/USD");
+  const data = await res.json();
+  exchangeRates = data.rates;
+  const currenciesList = Object.keys(exchangeRates);
+  console.log(exchangeRates);
+  console.log(currenciesList);
+
+  const currencyFromDropdown = document.getElementById("currency-from");
+  const currencyToDropdown = document.getElementById("currency-to");
+
+  currenciesList.forEach((currency) => {
+  const option = document.createElement("option");
```

Comment:
For having a clear and readable code, it is a good idea to format your code before submitting. 
Here is how to do it in VS code: https://code.visualstudio.com/docs/editor/codebasics#_formatting